playbooks/ansible-test-cloud-integration-base/pre.yaml

```diff
@@ -1,14 +1,6 @@
 ---
 - hosts: controller
   tasks:
-    - name: Setup tox role
```

Comment:
We still need this

Reply:
So, I'd like to keep this for now, for our http check in vmware. As, the next step I would like to do, is refactor

playbooks/ansible-test-cloud-integration-base/run.yaml
playbooks/ansible-test-network-integration-base/run.yaml 

into a shared playbook.

While I agree, this change seems to make sense here, it will not in network. As network cannot talk to appliances today.